packages/admin/cms-admin/src/generator/future/generateForm/generateFormField.ts

```diff
@@ -43,7 +51,7 @@ export function generateFormField({
     const readOnlyPropsWithLock = `${readOnlyProps} ${endAdornmentWithLockIconProp}`;
 
     const imports: Imports = [];
-    let formValuesConfig: GenerateFieldsReturn["formValuesConfig"] = [];
+    let formValuesConfig: GenerateFieldsReturn["formValuesConfig"] = []; // FormFields should only contain one entry
```

Comment:
because formLayout fieldSet needs this. fieldSet does only wrap and not modify any of it's wrapped fields. (I'm currently implementing "dependent async-select", this does also benefit of this)